### PR TITLE
Migrate source code

### PR DESCRIPTION
**Pull request by SimonWren**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,21 +8,32 @@ The various configuration frameworks are intialised by supplying appsetting.json
...
 
 ```
-[Binding]
+[   [Binding]
     public class ContactUsConfigurationSetup
     {
         private readonly ScenarioContext _context;
         private readonly ObjectContext _objectContext;
         //private readonly IConfigurationRoot _configurationRoot;
+        //private readonly IConfigSection _configSection;
```
Comment (jbarnes16503) [inline README.md:17]: Did you leave this comment in intentionally?

File: README.md [hunk 2]
```diff
@@ -8,21 +8,32 @@ The various configuration frameworks are intialised by supplying appsetting.json
 # Example Usage
 
 ```
-[Binding]
+[   [Binding]
```
Comment (jbarnes16503) [inline README.md:11]: Looks like a syntactic error here. Does this still build?!

File: README.md (hunk 1 above)
Comment (SimonWren) [reply README.md]: removed

File: README.md (hunk 2 above)
Comment (SimonWren) [reply README.md]: it's in the readme so doesn't impact the build.  but fixed :-)

**Pull request merged**